apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/contacts/ContactListNavView.kt

```diff
@@ -0,0 +1,112 @@
+package chat.simplex.common.views.contacts
+
+import androidx.compose.runtime.*
+import androidx.compose.ui.graphics.Color
+import dev.icerock.moko.resources.compose.painterResource
+import dev.icerock.moko.resources.compose.stringResource
+import chat.simplex.common.model.*
+import chat.simplex.common.platform.*
+import chat.simplex.common.views.chat.*
+import chat.simplex.common.views.chat.item.ItemAction
+import chat.simplex.common.views.chatlist.*
+import chat.simplex.common.views.helpers.*
+import chat.simplex.res.MR
+import kotlinx.coroutines.delay
+
+@Composable
+fun ContactListNavLinkView(chat: Chat, nextChatSelected: State<Boolean>) {
+    val showMenu = remember { mutableStateOf(false) }
+    val disabled = chatModel.chatRunning.value == false || chatModel.deletedChats.value.contains(chat.remoteHostId to chat.chatInfo.id)
+    LaunchedEffect(chat.id) {
+        showMenu.value = false
+        delay(500L)
+    }
+    val selectedChat = remember(chat.id) { derivedStateOf { chat.id == chatModel.chatId.value } }
+
+    when (chat.chatInfo) {
+        is ChatInfo.Direct -> {
```

Comment:
for contact cards (e.g. "SimpleX Chat team" - and we plan to add 1 or 2 more) connectContactViaAddress alert should be shown on new chat sheet, same as contact requests. Also probably both contact cards and contact requests should be colored in accent color to differentiate from contacts and show they lead to some other action than simply opening